serving/samples/helloworld-nodejs/README.md

```diff
@@ -92,17 +92,14 @@ recreate the source files from this folder.
    # Create and change to the app directory.
    WORKDIR /usr/src/app
 
-   # Copy application dependency manifests to the container image.
-   # A wildcard is used to ensure both package.json AND package-lock.json are copied.
-   # Copying this separately prevents re-running npm install on every code change.
-   COPY package*.json ./
```

Comment:
You are removing a build optimization.

Reply:
Let's just close this. We left this off with the realization that there is no way to make a docker file that solves edge cases and is easy to consume in the context of hello world. This docker file will produce broken images if the node_modules tree has a native dependency. It needs an ignore file to omit the users local node_modules from the `copy . .` But ignore files are complex as we've discussed before.

Reply:
It npm installs then copies the local files into the image including local node_modules like it's doing all the work then undoing it.